cmd/sops/main.go

```diff
@@ -666,11 +666,19 @@ func main() {
 		},
 		cli.StringFlag{
 			Name:  "unencrypted-regex",
-			Usage: "set the unencrypted key suffix. When specified, only keys matching the regex will be left unencrypted.",
+			Usage: "set the unencrypted key regex. When specified, only keys matching the regex will be left unencrypted.",
```

Comment:
This is unrelated, but I noticed it and fixed.

Reply:
Could you please move all unrelated fixes to another PR?

Reply:
Done: https://github.com/getsops/sops/pull/1300